go.mod

```diff
@@ -20,7 +19,6 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
-	github.com/swaggo/swag v1.7.3
```

Comment:
I think you need to run "go mod tidy" and commit the changes, otherwise "make all" won't run